js/utils/CommonUtils.js

```diff
@@ -0,0 +1,231 @@
+define(
+  (require, factory) => {
+    const d3 = require('d3');
+		const _ = require('lodash');
+		const ko = require('knockout');
+
+    const mapConceptData = function (data) {
```

Comment:
Most of methods in the class are for smth like ChartUtils, not project-wide utils